examples/energy_mng/src/controllers/weather.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+use crate::ui::*;
+use chrono::prelude::*;
+use core::cmp::Ordering;
+use slint::*;
+use std::rc::Rc;
+use weer_api::*;
+
+const WEATHER_API_KEY: &str = "WEATHER_API";
+const LAT_BERLIN: f32 = 52.520008;
+const LONG_BERLIN: f32 = 13.404954;
+const FORECAST_DAYS: i64 = 5;
+
+pub fn setup(window: &MainWindow) {
+    if let Some(api_key) = std::env::var_os(WEATHER_API_KEY) {
```

Comment:
Please add a README that documents this and how to get an API key.
(could also add a println! if the env variable is not set to explain it could be set)

Reply:
Sure

Reply:
I was wondering as well.I think it should be both:
```rust
option_env!(WEATHER_API_KEY").or_else(||std::env::var_os( "WEATHER_API_KEY"))
```

Note that i'm afraid this code path will not work in MCU environment because the crate that fetch the weather rely on some http stuff that is probably not available as is there

Reply:
For mcu's maybe it will be necessary to have a server that run e.g. on laptop and send the data to the mcu.